database/seeds/UserBeatmapSeeder.php

```diff
@@ -0,0 +1,350 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class UserBeatmapSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        try {
+            DB::table('phpbb_user_group')->delete();
+            DB::table('osu_scores')->delete();
+            DB::table('osu_scores_high')->delete();
+
+            DB::table('phpbb_users')->delete();
+
+            $datapath = base_path().'/database/data/json/';
+        // USERS
+
+        // Store saved / overwritten data in arrays
+        $savedusers = [];
+            $overusers = [];
+            $overstats = [];
+            $overhist = [];
+            $overscores = [];
+            $overevents = [];
+            $savedbeatmaps = [];
+            $savedscores = [];
+            $overbeatmaps = [];
+            $savedbeatmapsets = [];
+            $overbeatmapsets = [];
+            $error_log = [];
+            $exceptions = [];
+            $saveduserids = [];
+
+            $users = json_decode(file_get_contents($datapath.'users.json'));
```

Comment:
You moved user generating from the model factory to a seeder, but you forgot to change the tests appropriately.

Also, shouldn't we have a model factory to generate random clean users for unit testing? Picking an user from database at random seems unreliable.